ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh

```diff
@@ -101,15 +103,15 @@ export MAKEFILE_TARGET='setup test_parallel'
 export PROVIDER_IMAGE={{ PROVIDER_IMAGE }}
 {% endif %}
 
-{% if PROVIDER_IMAGE != ASSISTED_CONTROLLER_IMAGE %}
+{% if HYPERSHIFT_IMAGE != ASSISTED_CONTROLLER_IMAGE %}
 export HYPERSHIFT_IMAGE={{ HYPERSHIFT_IMAGE }}
 {% endif %}
 
 {% if JOB_TYPE == "presubmit" and REPO_NAME == "assisted-service" %}
 export SERVICE_BRANCH={{ PULL_PULL_SHA }}
 {% endif %}
 
-{% if REPO_NAME != "assisted-service" %}
+{% if JOB_TYPE != "presubmit" or  REPO_NAME == "release" %}
```

Comment:
nit: an extra space in there

btw, I do not recall what we originally intended with this code, but with this change (and with the previous code) we are having different images in assisted-service presubmit jobs and in rehearsals of those jobs
Are we able to fix it in some way? (maybe there's some reference to which repo we're rehearsing in the prow env?)

Reply:
:eagle:

It was here in the old script: https://github.com/openshift/release/blob/e6651ea1fbbb43e98c294b39098aa2d407680cd1/ci-operator/step-registry/assisted/baremetal/setup/assisted-baremetal-setup-commands.sh#L146-L158

and you approved https://github.com/openshift/release/pull/18291 :P

Reply:
but, yes I don't understand why we do that

Reply:
may be let's keep this problem out of this PR, I will open a ticket to address the issue separately. I would first like to fix the failing capi jobs.

Reply:
sure no problem
thanks!